pallets/automation-time/src/exchange.rs

```diff
@@ -0,0 +1,143 @@
+// This file is part of OAK Blockchain.
```

Comment:
pls explain name of file `exchange.rs`

Reply:
I am bad at names....

The point of that file is to hold the trait we want our automation pallets to use to interact with Balances. This includes fees and transfers. At some point I will extract this into another util lib so it can used between pallets.